styles/highlight-selected.less

```diff
@@ -14,10 +14,45 @@
     .highlight-selected .region {
       border-radius: 3px;
       box-sizing: border-box;
-      background-color: transparent;
+      background-color: white;
       border-width: 1px;
       border-style: solid;
     }
+    .highlight-selected.selection1{
+      .region{
+        background-color: red
+      }
+    }
+    .highlight-selected.selection2{
+      .region{
+        background-color: orange
+      }
+    }
+    .highlight-selected.selection3{
+      .region{
+        background-color: yellow
+      }
+    }
+    .highlight-selected.selection4{
+      .region{
+        background-color: green
+      }
+    }
+    .highlight-selected.selection5{
+      .region{
+        background-color: blue
+      }
+    }
+    .highlight-selected.selection6{
+      .region{
+        background-color: indigo
+      }
+    }
+    .highlight-selected.selection7{
```

Comment:
I appreciate your comments and I shared them with my group!  By the way,
for over 7 selections, the default is white

On Mon, May 9, 2016 at 5:51 PM, Richard Race <notifications@github.com>
wrote:

> In styles/highlight-selected.less
> <https://github.com/richrace/highlight-selected/pull/124#discussion_r62579453>
> :
>
> > +    .highlight-selected.selection4{
> > +      .region{
> > +        background-color: green
> > +      }
> > +    }
> > +    .highlight-selected.selection5{
> > +      .region{
> > +        background-color: blue
> > +      }
> > +    }
> > +    .highlight-selected.selection6{
> > +      .region{
> > +        background-color: indigo
> > +      }
> > +    }
> > +    .highlight-selected.selection7{
>
> What happens if I have more than 7 selections?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/richrace/highlight-selected/pull/124/files/9f522169f5f71cbb9711d4de9132a40a17653f09#r62579453>
>